tls/test_tls_basic.py

```diff
@@ -53,22 +40,99 @@ def start_all(self):
         self.assertTrue(deproxy_srv.wait_for_connections(timeout=1),
                         "No connection from Tempesta to backends")
 
+class TlsBasic(TlsBasicBase):
+
+    backends = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' :
+                'HTTP/1.1 200 OK\r\n' \
+                'Content-Length: 20000\r\n' \
+                'Connection: keep-alive\r\n\r\n'
+                + 'x' * 20000
```

Comment:
I have to raise question about side effects once again. There was a basic test about https: send a small request and expect a correct response, a client connection is closed  when a error happens or when a client asks it. It's just a very basic test just to prove, that TLS works. Then you have test failures in more complicated tests, this one tells how far things are broken: TLS is fully broken or only _some_ scenarios. 

But in this patch  you're converting that simple scenario into very specific case about handling connection close on TLS when a huge message is being sent.   It's my fault that the original test had no documentation. But we heavily rely on very subtile and specific cases here, and none of them was documented. That's not good, nobody is going remember what is tested, and what is not. 

So we need here a separate test case on faulty scenario. 

Next, Do we really reproduce the faulty scenario here? I've never seen the issue with 2Kb files, at least 10kb was required to get stable failures. On remote scenario, i.e. client and Tempesta on different VM. In local scenario even 10kb is too low to reproduce the issue reliably. So the test won't be helpful on CI. Probably we can set fixed receive window size on client socket via `setsockopt()` to make the test more reliable without using huge files...